src/sync-controller.js

```diff
@@ -49,9 +49,9 @@ export const syncPointStrategies = [
           let segmentStart = segmentTime + syncController.datetimeToDisplayTime;
           let distance = Math.abs(currentTime - segmentStart);
 
-          // Once the distance begins to increase, we have passed
+          // Once the distance begins to increase, or if distance is 0, we have passed
           // currentTime and can stop looking for better candidates
-          if (lastDistance !== null && lastDistance < distance) {
+          if (lastDistance !== null && (distance === 0 || lastDistance < distance)) {
```

Comment:
This is a legitimate fix, the previous code assumed that if the `currentTime` started on the `segmentStart` that we should include that as a valid syncPoint. From my testing this is not the case. We should instead exclude syncPoints that would start exactly on `currentTime`.

EX: `segment#16` at `170s` and `currentTime` at `170s` should choose `segment#15` rather than `segment#16` as `currentTime` may need data from `segment#15` and `segment#16` to play.

Reply:
This explanation would be good to put in the commit body when we merge.